client/post-editor/editor-ground-control/style.scss

```diff
@@ -38,15 +38,38 @@
 	margin: 0 4px 8px;
 }
 
-.editor-ground-control__publish-button {
+.editor-ground-control .editor-publish-button {
+	@extend .is-primary;
+
 	border-radius: 4px 0 0 4px;
 	flex-grow: 1;
+
+	@include breakpoint( "<660px" ) {
+		display: none;
+	}
 }
+
 .editor-ground-control__time-button {
-	border-radius: 0 4px 4px 0;
+	@include breakpoint( ">660px" ) {
+		@include is-primary;
```

Comment:
>This is awful. If you have a better alternative _[...]_

It doesn't seem that bad to me, to be honest. Seems the primary styling comes more as a consequence of its positioning adjacent the publish button, not that the date picker button is itself a primary action. However, in terms of implementation, instead of creating a separate mixin, could we simply change this to `@extend .button.is-primary;` ?